allennlp/predictors/predictor.py

```diff
@@ -58,6 +65,88 @@ def predict_json(self, inputs: JsonDict) -> JsonDict:
         instance = self._json_to_instance(inputs)
         return self.predict_instance(instance)
 
+    def inputs_to_labeled_instances(self, inputs: JsonDict) -> List[Instance]:
```

Comment:
Yeah, this is a shortcut method to save a few lines of code, so you just call this instead of the three lines inside the method.  I'm not certain it's worth adding a new public API call for this.  If it is, using `json` instead of `inputs` (matching `json_to_instance`) is definitely better.

Reply:
yeah not sure if adding is better or not. We use it everywhere. ill change to `json_to_labeled_instances` for now.